microsoft-365/compliance/search-the-audit-log-in-security-and-compliance.md

```diff
@@ -1127,7 +1127,7 @@ See the [Audited activities](#audited-activities) section in this article for a
 
 **How long does it take for an auditing record to be available after an event has occurred?**
 
-Most auditing data is available within 30 minutes but it may take up to 24 hours after an event occurs for the corresponding audit log entry to be displayed in the search results. See the table in the [Before you search the audit log](#before-you-search-the-audit-log) section of this article that shows the time it takes for events in the different services to be available.
+Most auditing data is available within 60 to 90 minutes but it may take up to 24 hours after an event occurs for the corresponding audit log entry to be displayed in the search results. See  [Before you search the audit log](#before-you-search-the-audit-log) section of this article that shows the time it takes for events in the different services to be available.
```

Comment:
```suggestion
Most auditing data is available within 60 to 90 minutes but it may take up to 24 hours after an event occurs for the corresponding audit log entry to be displayed in the search results. See [Before you search the audit log](#before-you-search-the-audit-log) section of this article that shows the time it takes for events in the different services to be available.
```